app/models/line_item.rb

```diff
@@ -0,0 +1,19 @@
+class LineItem < ApplicationRecord
+  ### ASSOCIATIONS ###
+
+  belongs_to :order
+  belongs_to :meal
+  has_many :ingredients_meals, through: :meal
+
+  ### CALLBACKS ###
+
+  after_commit :update_order_price
+
+  def price
```

Comment:
instead of this add a column on line item and save price there.

Reply:
Done